src/System.IO.FileSystem/src/System/IO/UnixFileSystem.cs

```diff
@@ -13,6 +13,8 @@ namespace System.IO
     /// <summary>Provides an implementation of FileSystem for Unix systems.</summary>
     internal sealed partial class UnixFileSystem : FileSystem
     {
+        internal const int DefaultBufferSize = 4096;
```

Comment:
I'll try and find a good place for it in another PR.  I might create an InternalConstants class in IO- thoughts?